bsp/tm4c123bsp/libraries/Drivers/drv_flash/drv_flash_f1.c

```diff
@@ -0,0 +1,197 @@
+/*
```

Comment:
Is F1 series also available in TI MCU?

Reply:
Sorry,I ignored it.  Now I have removed it.